### PR TITLE
Add a better version of world countries map

### PR DESCRIPTION
**Pull request by neveldo**
Here is a better vesion of the world map based on https://commons.wikimedia.org/wiki/File:BlankMap-World6-Equirectangular.svg . 

I have made another version that includes antartica. @Indigo744 , do you think we should replace your world map with antartica by it in order to make them both uniform ?

I think I still need to double check the countries list and the translation from GPS coordinates to cartesian ones.

### CODE REVIEW

**Comment by neveldo** [comment]
Here is a JSFiddle example (I just forked the one that you have done for the map with antartica ;) ) : https://jsfiddle.net/neveldo/kL7o17cp/

File: world/world_countries.js [hunk 1]
```diff
@@ -0,0 +1,230 @@
+/*!
+ *
+ * Jquery Mapael - Dynamic maps jQuery plugin (based on raphael.js)
+ * Requires jQuery and Mapael >=2.0.0
+ *
+ * Map of World countries with Antarctica
```
Comment (Indigo744) [inline world/world_countries.js]: "Without Antarctica", no?

File: world/world_countries.js [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+
+    "use strict";
+    
+    $.extend(true, Mapael,
+        {
+            maps :  {
+                world_countries : {
+                    width : 999.29852,
+                    height : 392.03476,
+                    getCoords : function (lat, lon) {
```
Comment (Indigo744) [inline world/world_countries.js]: Should we still use comma-first style?

**Comment by Indigo744** [comment]
I agree with you, you should add the one with antarctica so they are similar. However, could you make sure the map is viewable to all coordinates (from -90° to 90° latitude to -180 to 180° longitude)?

File: world/world_countries.js (hunk 1 above)
Comment (neveldo) [reply world/world_countries.js]: Well seen, I fix it

File: world/world_countries.js (hunk 2 above)
Comment (neveldo) [reply world/world_countries.js]: You are right, it comes from the getCoords.php tool on mapael-documentation repository, I will make a PR to fix it.

**Comment by neveldo** [comment]
My bad, the map seems not to be viewable to strictly all coordinates :( Maybe we should try to find a better source ?

I have made the adjustements anyway, and I have added 'world_countries_with_antarctica_2.js' , but don't know if they worth to be merged.

(that said, I wonder if a map that cover longitudes from -90° to 90° is the best suitable for the regular visualizations, as the Russia country is split into two parts in this case)

**Comment by Indigo744** [comment]
Maybe then we should rename the existing world map with Antarctica as "full_long_and_lat" (it would be wrong to completely replace this one, as it was requested by a user).

Your new maps could then have a proper name and be in sync.

**Comment by neveldo** [comment]
Of course the world_countries_with_antarctica that you have built is very useful to add points at the extreme latitudes and longitudes.

I have updated the PR by renaming world_countries_with_antarctica to world_countries_with_antarctica_full_long_and_lat and world_countries_with_antarctica_2 to world_countries_with_antarctica.

Otherwise, another alternive is to use one of the miller or the mercator projection of the map of the world you have made few times ago as default world map. What do you think about this solution ? (I haven't checked if they are cover the full range of latitudes and longitudes)

**Comment by neveldo** [comment]
I have just checked the miller and the mercator projections, and they don't cover the full range of latitude and longitudes either

**Comment by Indigo744** [comment]
Well, Miller and Mercator projections diverge near the pole, so the +90 / -90 can not be mapped on them.

No I think we can keep it that way, with different maps so user can choose 😃

**Comment by Indigo744** [comment]
I've made a diff between your new map (`world_countries_with_antarctica`) and mine (`world_countries_with_antarctica_full_long_and_lat`), and there are 177 in yours, and 166 in mine.

`world_countries_with_antarctica`: 177 countries
`world_countries_with_antarctica_full_long_and_lat`: 166 countries
The ones that are missing are: `BS`, `FJ`, `FK`, `FO`, `KM`, `NC`, `PF`, `PS`, `SB`, `TL`, `VU`, `WS`

Now regarding your new map (`world_countries_with_antarctica`) and the old one (`world_countries`), we have the same number of countries (177), but some differences in naming.
I'll update this comment when I'm done with identifying the differences.

But I think we should try (at least for the new map world_countries) to be as much as possible coherent with the previous one.

**Comment by neveldo** [comment]
Thanks for having checked the diff of countries between the two maps, I will check them and update the maps !

**Comment by neveldo** [comment]
`AQ` is for antarctica (it can be found within world_countries_with_antarctica but not in world_countries), that is ok.

`FO` is for Faroe Islands (https://en.wikipedia.org/wiki/ISO_3166-2:FO), `KM` is for Comoros : https://en.wikipedia.org/wiki/ISO_3166-2:KM, `PF` is for french polynesia (https://en.wikipedia.org/wiki/ISO_3166-2:PF) and `WS` is for Samoa. I will keep these ISO codes as it seems that they were missing in the previous version of the map.

`TD`, `SN`, `NA`, `KG` seems to be present in both old and new version of the old map. So I have made another diff between the old and the new map, and I found these four countries that are in the old map but not in the new one : KO, NCY, SOL, TF. It seems that the old world map contains some country that are not yet official recognized by the international community :

- 'KO' is for Kosovo. It seems that this country isn't recognized officialy by everybody for now. According to the ISO 3166-2, KO is not assigned. In the new world map, The Kosovo is part of the Serbia. Do you think we should split them appart in order to be compliant with the old map, or follow the codes from ISO 3166-2 ?

- 'NCY' :In the same way, it seems that NCY stands for the northen part of the Cyprus island that is not yet officialy recognized.

- 'SOL' : SOL is for the Somaliland country (a part from Somalia) that is not yet officialy recognized.

- 'TF' : it's the "French Southern and Antarctic Lands", it seems they are part of France, but they have an official ISO code. They are so tiny that they have been removed by the simplification process. (However, in the new map, these islands are linked with antarctica, and some of them aren't present).I wonder if we should still add these island ?

**Pull request merged**